python/perf-kernels/rmsnorm.py

```diff
@@ -16,6 +16,14 @@ def is_hip():
     return triton.runtime.driver.active.get_current_target().backend == "hip"
 
 
+if torch.cuda.is_available():
+    current_device_index = torch.cuda.current_device()
+    current_device = torch.cuda.get_device_properties(current_device_index)
+    NUM_SMS = current_device.multi_processor_count
```

Comment:
Can you put this in a function instead of a global?

Reply:
not sure where we should have this function call, If we put in the wrapper, it will produce overhead for this function call.

Reply:
Hmm....we can't use a global since this would be imported into a larger e2e workload. Can this be passed in instead?

Reply:
done